proposals/kroxylicious-operator-api-alternative.md

```diff
@@ -223,35 +232,19 @@ spec:
 
   # Points to the cluster being proxied.  Can either be Strimzi Kafka resource or endpoint details.
   targetCluster:
-    # one of: resourceRef or bootstrapping
-    resourceRef:
-      kind: Kafka  # must be Kafka
-      group: strimzi.io # must be strimzi.io
+    clusterRef:
+      kind: Kafka|TargetClusterRef  # must be Kafka or TargetClusterRef
```

Comment:
the examples later on imply that we have default values for `kind` and `group`, defaulting to TargetClusterRef defails. We should doc that somewhere.